server/difido-server/src/main/java/il/co/topq/report/front/rest/ExecutionResource.java

```diff
@@ -65,6 +67,30 @@ public int post(ExecutionDetails executionDetails) {
 
 	}
 
+	@POST
+	@Path("/update")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.TEXT_PLAIN)
+	public String updateExecutionMetadata(ExecutionMetadataUpdateRequest updateRequest) {
```

Comment:
I didn't find a PUT method which supports a **query string**.
Found only this one...

```
**
* Used to update that a single execution should not be active any more.
* This is Irreversible.
* 
* @param executionIndex
*            the id of the execution
* @param active
*            - Set to not active
* @param locked
*            - Set the execution to locked. Will no be deleted
* 
*/
@PUT
@Path("/{execution: [0-9]+}")
public void put(@PathParam("execution") int executionIndex,
@QueryParam("active") Boolean active, @QueryParam("locked") Boolean locked) {
.
.
.
}
```
Should I make changes to this method?